Dockerfile

```diff
@@ -70,9 +70,10 @@ ARG VITE_BUILDTIME
 ARG VITE_RELEASE
 ARG VITE_COMMITHASH
 ARG VITE_IDLE_TIMEOUT_IN_MS
+ARG NODE_OPTIONS
 
 # Fix FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
-ENV NODE_OPTIONS="--max-old-space-size=8192"
+ENV NODE_OPTIONS=${NODE_OPTIONS}
```

Comment:
The underlying problem https://github.com/vitejs/vite/issues/2433 could be linked here or somewhere else where it would be a good place, so that it's known why this is needed, and perhaps not needed in the future when the underlying problem is solved.

Reply:
Added